go.mod

```diff
@@ -16,9 +16,9 @@ require (
 	github.com/stretchr/testify v1.6.1
 	golang.org/x/net v0.0.0-20200202094626-16171245cfb2
 	gopkg.in/ini.v1 v1.57.0 // indirect
-	k8s.io/api v1.17.8
-	k8s.io/apimachinery v1.17.8
-	k8s.io/client-go v1.17.8
+	k8s.io/api v0.17.8
```

Comment:
@jessesuen I think you were partly correct, I think this is the issue with the `go.mod` file

Reply:
@alexec I don't know why, but `go mod tidy` changes this back. I don't think this is a problem.

Reply:
@alexec I think you are right, `v1.17.8` is incorrect. I've just ran into the same issue trying to update to the latest gitops-engine. https://github.com/argoproj/gitops-engine/pull/113